update_docker_images_txt.py

```diff
@@ -0,0 +1,74 @@
+"""Update ./docker_images.txt."""
+
+import argparse
+import logging
+
+
+def main() -> None:
+    """Prep and execute Condor job.
+
+    Make scratch directory and condor file.
+    """
+    parser = argparse.ArgumentParser(
+        description=("Update the ./docker_images.txt"),
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+    )
+
+    parser.add_argument(
+        "--action",
+        required=True,
+        choices=["add", "remove"],
+        help="What do you want to do with the docker images?",
+    )
+    parser.add_argument(
+        "--docker-tag",
+        required=True,
+        help="The docker image tag",
+    )
+    parser.add_argument(
+        "--dest-dir",
+        required=True,
+        help="The destination directory, eg: realtime",
+    )
+    parser.add_argument(
+        "--remove-docker-repo",
+        default=False,
+        action="store_true",
+        help="whether to remove the docker image's repo when inserting to CVMFS dir",
+    )
+
+    args = parser.parse_args()
+    for arg, val in vars(args).items():
+        logging.warning(f"{arg}: {val}")
+
+    # TAG  = "icecube/skymap_scanner:3"
+    # LINE = "docker://icecube/skymap_scanner:3 realtime/skymap_scanner:3"
+
+    if args.remove_docker_repo:
+        dest_file = args.docker_tag.split("/", max_split=1)[1]
+    else:
+        dest_file = args.docker_tag
+
+    line_to_add = f"docker://{args.docker_tag} {args.dest_dir}/{dest_file}"
```

Comment:
Do a proper path join here, in case `dest_dir` is empty (so there shouldn't be a `/` in that case).